src/routes/Dashboard/components/Map/Map.style.json

```diff
@@ -0,0 +1,116 @@
+[{
```

Comment:
What would be the alternative? It is the style attribute of the Google Maps object. Having it in the code itself was not good for readability. This makes it easy to replace with another style.